### ISSUE TITLE
Phuc - record 20-minute demo of your proposed workshop

### ISSUE THREAD

**Issue by kalyanoliveira** [USFGDSC/.github#182]
## General assignment description

For this assignment (deadline: 1 week; so that would be August 7th), I'll be asking workshop leads to record a 20-minute demo of their proposed workshop.

Find your proposed workshop, which was previously submitted in https://github.com/USFGDSC/.github/issues/165, attached [here](https://github.com/user-attachments/files/21538019/phucs-workshop-final-summary.pdf).

Should any questions, time constraint concerns, or anything else arise, please do not hesitate to contact me either through here or our usual communication platforms.

## Requirements

- Prepare a continuous, 20-minute *segment* of your workshop. That means that you can record something that's supposed to happen in beginning of your workshop, in its ending, in the middle; doesn't really matter.

- Create a screen-capture with your own audio and slides (strongly recommended, the latter, but if you deem them unnecessary, that is fine also) of that 20-minute segment (suggested tool: OBS).

- Upload your recording here (if the file size is too large, then attempt to provide a link to the recording. If that doesn't work either, we can try to figure out something else).

Following these recordings, leads are to reflect upon their current development status, iterate, and improve their workshops.

**Comment by kalyanoliveira** [comment]
Hey Phuc, since you could not attend our meeting right now, please report your current progress here, and if you have any concerns or difficulties, let me know too.

**Comment by kalyanoliveira** [comment]
@lenguyenduyphuc how are you doing with this assignment? I had posted the deadline for yesterday.

**Comment by lenguyenduyphuc** [comment]
https://usfedu.sharepoint.com/:v:/t/AIPGRP/EYY9A09dpY9KrWpNe3_rc14B4_ObjnxTJ7UMkG1dYUklgA?nav=eyJyZWZlcnJhbEluZm8iOnsicmVmZXJyYWxBcHAiOiJTdHJlYW1XZWJBcHAiLCJyZWZlcnJhbFZpZXciOiJTaGFyZURpYWxvZy1MaW5rIiwicmVmZXJyYWxBcHBQbGF0Zm9ybSI6IldlYiIsInJlZmVycmFsTW9kZSI6InZpZXcifX0%3D&e=jHdmVE
 @kalyanoliveira  This is recording of myself. Only 5 minutes because I actually do not know how to do it correctly and this is my try. Please give feedback to me.

**Comment by kalyanoliveira** [comment]
Just some quick bullet points as feedback for now (before our meeting today): 

- Make sure to take some time to explain concepts in AI (ML/DL). Training, overfitting, datasets, labels, fine-tuning, models; this kind of language is familiar to some, but not to others. This comes from the need to make our workshops more tools-based rather than project-based!

- Decide what to do during the 15 minutes of training. Will you be the only person training? Will you give participants a choice to do so? If so, what will you do whilst they are waiting? That kind of stuff.

- Jupyter with colab runs on the cloud, so that gives participants equal computing power, which you should account for in the time planning of your workshop. But what about download speeds? Are the datasets too large? Make sure to also plan for that.

Overall, these keypoints are more about the specific process of your workshop. If you manage to explain the key concepts involved in your project to those that are not that familiar, I think that we'll be on a good track with this workshop!
